env_file

```diff
@@ -20,7 +20,7 @@ case $region_choice in
 3) echo "::set-output name=region::eu-centeral-1" ;;
```

Comment:
How can i run this file in console?  env_file only for manuall runnning
![image](https://github.com/user-attachments/assets/bce4b8fd-9bd3-4f89-83ce-db41473c2d3d)